contracts/staking/StakingStorage.sol

```diff
@@ -79,8 +79,8 @@ contract StakingV1Storage is Managed {
 
     // -- Asset Holders --
 
-    // Allowed AssetHolders: assetHolder => is allowed
-    mapping(address => bool) public assetHolders;
+    // DEPRECATED: Allowed AssetHolders: assetHolder => is allowed
+    mapping(address => bool) public __DEPRECATED_assetHolders;
```

Comment:
We could make this variable private to avoid including the getter in the bytecode

Reply:
done 👍